project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+        apply_logic:
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+        costs: &costs_ashp_ducted_8-82_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3680.19 # for HSPF 8.82
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_HP 15
+
+      - &2_fed_min_mshp_ductless
+        option: HVAC Heating Efficiency|MSHP, SEER 14.5, 8.33 HSPF, Max Load # federal minimum is SEER 14.3, but 14.5 is lowest SEER in baseline ResStock
+        apply_logic:
+          - HVAC Has Ducts|No
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+        costs: &costs_mshp_ductless_8-33_HSPF
+        # Source: Custom regression by Brennan Less. Ductless heat pump project costs were regressed on nameplate tons,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ductless MSHP n=173, HSPF median 11, range 9.3–14.2. Implicitly includes multiple zones and electrical upgrades.
+        # Regression results in costs that are 15–30% higher than costs from The Heat Pump Store as documented in https://redwoodenergy.net/wp-content/uploads/2021/02/Pocket-Guide-to-All-Electric-Retrofits-of-Single-Family-Homes.pdf 
+        # LBNL data were not sufficient to include a relationship between SEER or HSPF and cost, so cost deltas for higher 
+        # and lower HSPF were added using the relationship between HSPF and cost for ductless MSHPs in REMDB: 2.2% addition or subtraction per point
+        # according to how many points above or below the HSPF rating is from 10.5 (https://remdb.nrel.gov/).
+          - value: 2679.52
+            multiplier: Fixed (1)
+          - value: 347.97
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_HP
+      
+    #### 1.3 Federal Minimum Level Heat Pump With Existing Heating Backup  ####
+      ##ASHP non-ducted heating##
+      - &fed_min_ashp_exist_backup_no_shared_ducts
+        option: HVAC Heating Efficiency|ASHP, SEER 15.05, 8.82 HSPF, Separate Backup
+        apply_logic: 
+          - not: HVAC Heating Efficiency|Shared Heating
+          - not: HVAC Cooling Efficiency|Shared Cooling
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Type|Non-Ducted Heating
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+        costs: *costs_ashp_ducted_8-82_HSPF
```

Comment:
This cost has electric backup, not the same

Reply:
Makes sense. Largeee use the same cost for HP with existing backup and HP with electric backup. 
#Cost Source in largee: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF. Shall we reach out to Brennan for the cost of HP with different backup

Reply:
Think so, let's note that in the intent doc